masterlist.yaml

```diff
@@ -8057,6 +8057,45 @@ plugins:
       - crc: 0xA082C1A4
         util: 'TES4Edit v4.0.3'
 
+  - name: 'Rathunas(01|-MidasMagic|-BetterMusicSystem)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/45880/'
+        name: 'Rathunas'
+  - name: 'Rathunas01.esp'
+    req:
+      - 'Cobl Main.esm'
+      - 'DLCShiveringIsles.esp'
```

Comment:
The requirements can be removed as well, they're master files.
![image](https://user-images.githubusercontent.com/1944639/141643700-fbe962a9-a53d-435d-9ec2-2a15f8e31a91.png)

```suggestion
```